### PR TITLE
update sample app and APIs of SDK

### PR DESCRIPTION
**Pull request by Abira16**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -253,24 +278,21 @@ getAuthorizationURL(Config).then((url) => {
...
-
 #### Description
 
 This method uses the authorization code and session state to send a request to the token endpoint to obtain the acess token and the id token. The sign-in functionality can be implemented by calling the getAuthorizationURL method followed by this method.
 
 #### Example
 
 ```TypeScript
-requestAccessTokenDetails(AuthUrl,Config).then((token)=>{
+requestAccessTokenDetails(AuthUrl).then((token)=>{
```
Comment (NipuniBhagya) [inline README.md:295]:
```suggestion
requestAccessTokenDetails(AuthUrl).then((token) => {
```

File: README.md [hunk 2]
```diff
@@ -382,7 +404,7 @@ const UserInfo =  await userInformation();
 ### revokeAccessToken
 
 ```TypeScript
-revokeAccessToken= async(config)
+revokeAccessToken= async()
```
Comment (NipuniBhagya) [inline README.md:407]:
```suggestion
revokeAccessToken = async()
```

File: README.md [hunk 3]
```diff
@@ -414,7 +436,7 @@ This method sends a refresh-token request and returns a promise that resolves wi
 #### Example
 
 ```TypeScript
-refreshAccessToken(Config).then((response)=>{
+refreshAccessToken().then((response)=>{
```
Comment (NipuniBhagya) [inline README.md:439]:
```suggestion
refreshAccessToken().then((response) => {
```

File: sample/Screen/LoginScreen.js [hunk 4]
```diff
@@ -15,106 +15,205 @@
...
+  initialize,
+  getAuthorizationURL,
+  requestAccessTokenDetails,
+  userInformation,
+  getDecodedIDToken,
+} from '@asgardeo/auth-react-native';
 import React from 'react';
-import {StyleSheet,View,Image, Text,Button} from 'react-native';
-import {initialize,getDataLayer,getAuthorizationURL} from '../src/authenticate'
+import {StyleSheet, View, Image, Text, Button, Linking} from 'react-native';
```
Comment (NipuniBhagya) [inline sample/Screen/LoginScreen.js:27]:
```suggestion
import { StyleSheet, View, Image, Text, Button, Linking } from "react-native";
```

File: sample/Screen/LoginScreen.js [hunk 5]
```diff
@@ -15,106 +15,205 @@
...
+  };
 
-    }
-  
-    render(){
+  render() {
     return (
       <View style={styles.mainBody}>
         <View>
+          <View style={styles.container}>
```
Comment (NipuniBhagya) [inline sample/Screen/LoginScreen.js:114]:
```suggestion
          <View style={ styles.container }>
```

File: sample/Screen/LoginScreen.js [hunk 6]
```diff
@@ -15,106 +15,205 @@
...
-    });
+      });
+  };
 
-    }
-  
-    render(){
+  render() {
     return (
       <View style={styles.mainBody}>
```
Comment (NipuniBhagya) [inline sample/Screen/LoginScreen.js:112]: There should be a space between the curly brackets and the characters. Please check and fix other places as well.

File: sample/Screen/LoginScreen.js [hunk 7]
```diff
@@ -15,106 +15,205 @@
...
+        </View>
       </View>
     );
-};}
+  }
+}
+
 export default LoginScreen;
 
 const styles = StyleSheet.create({
```
Comment (NipuniBhagya) [inline sample/Screen/LoginScreen.js:164]: Let's move these styles to a separate CSS file.

File: sample/Screen/SignIn.js [hunk 8]
```diff
@@ -1,152 +1,234 @@
-import React, {useEffect,useState} from 'react';
-import {Linking,Alert} from 'react-native';
-import {Page,Form,FormLabel,FormValue,Heading,ButtonContainer,Button} from '../components'
-import {requestAccessTokenDetails,SignOut,getSignOutURL,userInformation,refreshAccessToken,isAuthenticated,getAccessToken, getOIDCServiceEndpoints,getDecodedIDToken,revokeAccessToken}from '../src/authenticate'
-
+/**
+ * Copyright (c) 2020, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```
Comment (NipuniBhagya) [inline sample/Screen/SignIn.js:2]: Update the license year to 2021. Check other files as well.

File: sdk/.npmignore [hunk 9]
```diff
@@ -0,0 +1,12 @@
...
+npm-debug.log
+
+# Dependency directory
+node_modules
+
+# Runtime data
+tmp
+
+# Examples (If applicable to your project)
+examples
```
Comment (NipuniBhagya) [inline sdk/.npmignore:12]: Add a new line

**Review by NipuniBhagya — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (Abira16) [reply README.md]: Changed

**Pull request merged**